### PR TITLE
delete request  with body

### PR DESCRIPTION
**Pull request by liping86**
Ticket https://tickets.vivino.com/issues/37633

### CODE REVIEW

**Review by ssidletsky — APPROVED** [review]
LGTM

File: lib/wrest/native/delete.rb [hunk 1]
```diff
@@ -9,15 +9,15 @@
 
 module Wrest::Native
   class Delete < Request
-    def initialize(wrest_uri, parameters = {}, headers = {}, options = {})
+    def initialize(wrest_uri, body = '', headers = {}, parameters = {} ,options = {})
```
Comment (andreorvalho) [inline lib/wrest/native/delete.rb:12]: I would expect body to be an hash as well, why did you define it as an empty string for default?

**Review by andreorvalho — APPROVED** [review]
Seems great, just one question :)

File: lib/wrest/native/delete.rb (hunk 1 above)
Comment (liping86) [reply lib/wrest/native/delete.rb:12]: @andreorvalho  I'm just following the same pattern as the other http method with a body in this file like https://github.com/Vivino/wrest/blob/8747cc8988e2e9fdc0024d4363e8e86bb245fc0b/lib/wrest/uri.rb#L125 etc

Comment (andreorvalho) [reply lib/wrest/native/delete.rb:12]: yea I see. @corthmann do you know why is it like this?

**Pull request merged**